.gitignore

```diff
@@ -4,8 +4,6 @@
 # or operating system, you probably want to add a global ignore instead:
 #   git config --global core.excludesfile '~/.gitignore_global'
 
-# Ignore bundler config.
-/.bundle
```

Comment:
Shouldn't this be ignored in git?

Reply:
I've ended up porting across this

https://github.com/DFE-Digital/publish-teacher-training/blob/39f504be37a66c13f5abb4a2a44e7b3097a9a3af/.bundle/config